src/Tmds.Ssh/AesCtr.cs

```diff
@@ -0,0 +1,47 @@
+// This file is part of Tmds.Ssh which is released under MIT.
+// See file LICENSE for full license details.
+
+using System;
+using System.Security.Cryptography;
+
+namespace Tmds.Ssh;
+
+static class AesCtr
+{
+    public static byte[] DecryptCtr(ReadOnlySpan<byte> key, Span<byte> counter, ReadOnlySpan<byte> data)
```

Comment:
Let's accept the argument to decode to as a `Span` so the caller can manage the memory.

```
public static byte[] DecryptCtr(ReadOnlySpan<byte> key, Span<byte> counter, ReadOnlySpan<byte> data)
```
to
```
public static void DecryptCtr(ReadOnlySpan<byte> key, Span<byte> counter, ReadOnlySpan<byte> data` -> `ReadOnlySpan<byte> ciphertext, Span<byte> plaintext)
```